Update Set-CsTeamsCallingPolicy.md (#10460)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md

```

### -BusyOnBusyEnabledType

> Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, and Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: UserOverride setting is not currently available for use. As of now, this setting will be treated as disabled even if its is configured.

Reviewer: ```suggestion
Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, and Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: The userOverride setting is not currently available for use. The setting will be treated as disabled even if it is configured.
```
